I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 22, in script
    image bg_bar_scaled = ('background/bar.jpg', config.screen_width, config.screen_width)
Exception: Not a displayable: (u'background/bar.jpg', 1280, 720)

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 22, in script
    image bg_bar_scaled = ('background/bar.jpg', config.screen_width, config.screen_width)
  File "D:\Programs\RenPy\renpy-6.99.14-sdk\renpy\ast.py", line 965, in execute
    renpy.exports.image(self.imgname, img)
  File "D:\Programs\RenPy\renpy-6.99.14-sdk\renpy\exports.py", line 362, in image
    d = renpy.easy.displayable(d)
  File "D:\Programs\RenPy\renpy-6.99.14-sdk\renpy\easy.py", line 136, in displayable
    raise Exception("Not a displayable: %r" % (d,))
Exception: Not a displayable: (u'background/bar.jpg', 1280, 720)

Windows-8-6.2.9200
Ren'Py 7.0.0.196
Lonely Road 1.0
Mon Jul 30 21:02:07 2018
